client/src/app/containers/Game/ActionBox.jsx

```diff
@@ -0,0 +1,194 @@
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useInjectReducer, useInjectSaga } from 'redux-injectors';
+
+import Button from '@material-ui/core/Button';
+import ButtonGroup from '@material-ui/core/ButtonGroup';
+import CallSplitIcon from '@material-ui/icons/CallSplit';
+import FormControl from '@material-ui/core/FormControl';
+import FormGroup from '@material-ui/core/FormGroup';
+import Grid from '@material-ui/core/Grid';
+import Input from '@material-ui/core/Input';
+import InputLabel from '@material-ui/core/InputLabel';
+import Slider from '@material-ui/core/Slider';
+import SendIcon from '@material-ui/icons/Send';
+import ShuffleIcon from '@material-ui/icons/Shuffle';
+
+import { gameSaga } from './saga';
+import { sliceKey, reducer, actions } from './slice';
+import { selectCurrentAction, selectCurrentGame } from './selectors';
+
+const expNumCardsToCribForGame = game => {
+  if (game.teams.length === 3 || game.teams[0].players.length === 2) {
+    return 1;
+  }
+
+  return 2;
+};
+
+const ActionBox = props => {
+  useInjectReducer({ key: sliceKey, reducer: reducer });
+  useInjectSaga({ key: sliceKey, saga: gameSaga });
+
+  const dispatch = useDispatch();
+
+  const currentAction = useSelector(selectCurrentAction);
+  const activeGame = useSelector(selectCurrentGame);
+
+  function perc(value) {
+    return `${value}%`;
+  }
+
+  return (
+    <Grid item container justify='center' spacing={1}>
+      {props.phase === 'Deal' ? (
```

Comment:
💡 maybe we could split each phase's action box into a separate component to make this more readable